src/bookshelf/rebar.lock

```diff
@@ -53,45 +54,45 @@
  {<<"jsx">>,{pkg,<<"jsx">>,<<"2.9.0">>},1},
  {<<"lager">>,
   {git,"git://github.com/erlang-lager/lager.git",
-       {ref,"f26a8ea0f4c2ed6506a54e0b9890edd8148b9500"}},
+       {ref,"91575d4479c897451edf138c10fa86c3125ec52e"}},
   0},
  {<<"lhttpc">>,{pkg,<<"lhttpc">>,<<"1.6.2">>},1},
  {<<"meck">>,
   {git,"git://github.com/eproxus/meck.git",
-       {ref,"9753464ac1ebc0fc1a6777322bc871eaed82be35"}},
+       {ref,"06192a984750070ace33b60a492ca27ec9bc6806"}},
   0},
  {<<"mini_s3">>,
   {git,"git://github.com/chef/mini_s3.git",
-       {ref,"73c1be787dfe590113419091c531564e58592478"}},
+       {ref,"8a6b80ec60be799c5f69d9b2f22eb0af55b4dc77"}},
   0},
  {<<"mixer">>,
   {git,"git://github.com/chef/mixer.git",
        {ref,"58ded93d5c47675899d8e5e1589270f340ea66c5"}},
   0},
  {<<"mochiweb">>,
   {git,"git://github.com/mochi/mochiweb.git",
-       {ref,"cedc22f66a8e3f3885fb06babc0c0582418508f8"}},
+       {ref,"070594e4d66163d662ac7e3bfb75dadcc922dd7c"}},
   0},
  {<<"observer_cli">>,
   {git,"git://github.com/zhongwencool/observer_cli.git",
-       {ref,"bcc0dfaa2013a436ce7e597731acab6577321b43"}},
+       {ref,"baa70569bccc5508e9839e20768540ef3cdca016"}},
   0},
  {<<"opscoderl_wm">>,
   {git,"git://github.com/chef/opscoderl_wm.git",
        {ref,"7b222296b2f0eecd86f3c858f2185d21d55e8fa4"}},
   0},
  {<<"pooler">>,
-  {git,"git://github.com/chef/pooler.git",
-       {ref,"681c355abaacc5487ddf41a84b9ed53151a765fe"}},
+  {git,"git://github.com/seth/pooler.git",
```

Comment:
I think we want to be on chef/pooler instead of seth/pooler

Reply:
Why the heck.  I CHANGED that.  How did it get changed back?